metrics/lib/common.bash

```diff
@@ -42,6 +42,47 @@ function save_results(){
 	bash $LIB_DIR/send_results.sh -n "$1" -a "$2" -r "$3" -u "$4"
 }
 
+# This function checks existence of commands.
+# They can be received standalone standalone or as an array, e.g.
+#
+# cmds=(“cmd1” “cmd2”)
+# check_cmds "${cmd[@]}"
+function check_cmds()
+{
+	req_tools="$@"
```

Comment:
I have tested the advised changes in a pair of comments below and it did not work, however as I understand I just need to declare req_tool as local right?